karate-tests/src/test/java/patients/patientAccess/updatePatient.feature

```diff
@@ -216,4 +215,133 @@ Feature: Patient updates their details
     * match updatedPobCity == cityName
     * match parseInt(response.meta.versionId) == originalVersion + 2
 
- 
\ No newline at end of file
+  Scenario: Patient can add and remove nominated pharmacy
+
+    * def p9numberForPharmacy = '9446041481'
+    * def accessToken = karate.call('classpath:auth/auth-redirect.feature', {userID: p9numberForPharmacy, scope: 'nhs-login'}).accessToken
+    * def requestHeaders = call read('classpath:auth/auth-headers.js')
+    * configure headers = requestHeaders
+    * path 'Patient', p9numberForPharmacy
+    * method get
+    * status 200
+    * def originalVersion = parseInt(response.meta.versionId)
+    * def originalEtag = karate.response.header('etag')
+    # add nominated pharmacy
+
+    * configure headers = call read('classpath:auth/auth-headers.js') 
+    * header Content-Type = "application/json-patch+json"
+    * header If-Match = originalEtag
+    * path 'Patient', p9numberForPharmacy
+    * def requestBody = read('classpath:patients/requestDetails/add/nominatedPharmacy.json')
+    * def nominatedPharmacyUrl = requestBody.patches[0].value.url
+    * request requestBody
+    * method patch
+    * status 200 
+    * def idAftPharmacyAdd = response.meta.versionId
```

Comment:
The name of this variable doesn't sound right, you're not saving much by abbreviating After to Aft, better to be a bit more verbose but easier to read.